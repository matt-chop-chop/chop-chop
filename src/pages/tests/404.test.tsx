import { shallow } from "enzyme";
import React from "react";
import NotFound from "../404.page";

describe("NotFound", () => {
  it("is exported from NotFound", () => {
    expect(NotFound).toBeDefined();
  });

  it("renders as expected", () => {
    const wrapper = shallow(<NotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
