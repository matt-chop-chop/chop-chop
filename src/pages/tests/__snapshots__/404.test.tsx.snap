// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders as expected 1`] = `<NotFoundState />`;
